src/main/webapp/script.js

```diff
@@ -108,7 +109,9 @@ function createResultCard(name, address, photos, rating, totalCardCount) {
   const websiteButton = document.createElement('button');
   websiteButton.className = 'results-website-button';
   websiteButton.innerText = 'Visit Website';
```

Comment:
I would do this but you'll be able to much quicker since you already have this PR out. Let's make it so if the url passed is not for their personal website, the button text reads "Visit on Google Maps".

This should be as simple as adding an if statement to check whether the url returned starts with 'maps.google.com' if so, websiteButton.innerText = 'Visit on Google Maps" , otherwise, .....etc.

Let me know if you have any questions about this, I probably explained badly

Reply:
Sounds like a good idea to me.

Reply:
Just made this change and also created a different button message for when the website it unavailable